data/nomenclature.fab

```diff
@@ -6322,8 +6322,8 @@ Mars	378	_("Arimanes Rupes","rupes")	RU	-9.838700	212.304200	192.67213	_("Albedo
 Mars	1785	_("Elysium Rupes","rupes")	RU	25.244200	148.038800	140.49632	_("Albedo feature name.","origin")
 # TRANSLATORS: (Mars); From albedo feature at 70N, 248W.
 Mars	1363	_("Cydnus Rupēs","rupes")	RU	52.531500	112.209200	1550.8141	_("From albedo feature at 70N, 248W.","origin")
-# TRANSLATORS: (Mars); Named for calssical albedo feature at 55N, 260W.
-Mars	6261	_("Utopia Rupēs","rupes")	RU	43.532600	86.032600	2492.6814	_("Named for calssical albedo feature at 55N, 260W.","origin")
+# TRANSLATORS: (Mars); Named for classical albedo feature at 55N, 260W.
```

Comment:
It’s a sad, but these typos coming from outer database - I think we should collect the typos and write a report to mainstream again. Thanks for fix.